generate/operation.go.tmpl

```diff
@@ -12,34 +12,41 @@ func {{.Name}}(
     {{.GraphQLName}} {{.GoType.Reference}},
     {{end -}}
     {{end -}}
-) (*{{.ResponseName}}, error) {
-    {{- if .Input -}}
+) (*{{.ResponseName}}, {{if .Config.Extensions -}}map[string]interface{},{{end}} error) {
+    req := &graphql.Payload{
+        OpName: "{{.Name}}",
+        Query:  `{{.Body}}`,
+    {{if .Input -}}
     {{/* We need to avoid conflicting with any of the function's argument names
```

Comment:
This comment no longer applies, you can just remove it!

Reply:
done